.travis.yml

```diff
@@ -38,7 +38,7 @@ install:
     - pip install coverage nose pytest pytest-cov coveralls pycodestyle
     - pip install git+https://github.com/CEA-COSMIC/ModOpt.git
     - pip install git+https://github.com/AGrigis/pysphinxdoc.git
-    - pip install sphinx==2.2.1
+    - pip install sphinx==2.2.1 sphinxcontrib-bibtex
```

Comment:
I would prefer to have this on a new line, that way it is clearly visible of what are the things needed to build a doc.